cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsNetworkConnector.java

```diff
@@ -110,7 +110,11 @@ public CreatedCloudNetwork createNetworkWithSubnets(NetworkCreationRequest netwo
                 String cloudFormationTemplate = createTemplate(networkRequest, subnetRequests);
                 return createNewCfNetworkStack(networkRequest, credentialView, cloudFormationClient, cloudFormationTemplate, subnetRequests);
             } else {
-                throw new CloudConnectorException("Failed to create network.", e);
+                String region = networkRequest.getRegion().getRegionName();
+                String errorReason = awsCloudFormationErrorMessageProvider.getErrorReason(credentialView, region, cfStackName, ResourceStatus.CREATE_FAILED);
+                String message = String.format("Failed to create network: %s", errorReason);
+                LOGGER.debug(message, errorReason, e);
```

Comment:
Message is already formatted, so I don't think we need the errorReason here

Reply:
yeah, really. Thx, updated